.env.example

```
@@ -13,16 +13,26 @@ JWT_SECRET=your-super-secret-jwt-key-change-in-production-min-32-chars
 JWT_ACCESS_EXPIRES=15m
 JWT_REFRESH_EXPIRES=7d
 
-# AI Integration (Gemini — https://aistudio.google.com/apikey)
+# Provider selection
+# Use comma-separated list for automatic fallback (e.g. AI_PROVIDER=gemini,gemini)
+# STORAGE_PROVIDER options: supabase
+STORAGE_PROVIDER=supabase
+# AI_PROVIDER options: gemini
+AI_PROVIDER=gemini
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AI_PROVIDER key should go before the STORAGE_PROVIDER key

(UnorderedKey)